fds-lambda-locations-ingestion/src/main/java/com/provectus/fds/ingestion/LocationsHandler.java

```diff
@@ -0,0 +1,158 @@
+package com.provectus.fds.ingestion;
+
+import com.amazonaws.services.kinesis.producer.KinesisProducer;
+import com.amazonaws.services.kinesis.producer.KinesisProducerConfiguration;
+import com.amazonaws.services.kinesis.producer.UserRecordResult;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.KinesisEvent;
+import com.amazonaws.services.lambda.runtime.events.S3Event;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.event.S3EventNotification;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.model.S3Object;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.util.concurrent.FutureCallback;
+import com.google.common.util.concurrent.Futures;
+import com.provectus.fds.models.events.Location;
+import com.provectus.fds.models.utils.JsonUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class LocationsHandler implements RequestHandler<KinesisEvent, List<String>> {
+    private static final Logger logger = LoggerFactory.getLogger(LocationsHandler.class);
+
+    private static final String OK = "OK";
+    private static final String FAILED = "FAILED";
+
+    private static final String STREAM_NAME = "STREAM_NAME";
+    private static final String STREAM_NAME_DEFAULT = "locations";
+    private static final String AWS_REGION = "AWS_REGION";
+    private static final String AWS_REGION_DEFAULT = "us-west-2";
+
+    private static final String PREFIX = "locations/";
+    private static final String SEPARATOR = ",";
+
+    private final String streamName = System.getenv().getOrDefault(STREAM_NAME, STREAM_NAME_DEFAULT);
+    private final String region = System.getenv().getOrDefault(AWS_REGION, AWS_REGION_DEFAULT);
+
+    private final ObjectMapper mapper;
+    private final KinesisProducer producer;
+    private final FutureCallback<UserRecordResult> callback;
+    private final AtomicLong counter;
+
+    public LocationsHandler() {
+        counter = new AtomicLong();
+
+        mapper = new ObjectMapper();
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+        producer = new KinesisProducer(new KinesisProducerConfiguration()
+                .setRegion(region));
```

Comment:
what about other configs?

Reply:
I use default. Do we need to customize them through our CF template?

Reply:
Should have ability